mbedtls-sys/build/bindgen.rs

```diff
@@ -20,7 +19,21 @@ struct MbedtlsParseCallbacks;
 
 impl bindgen::callbacks::ParseCallbacks for MbedtlsParseCallbacks {
     fn item_name(&self, original_item_name: &str) -> Option<String> {
-        Some(original_item_name.trim_start_matches("mbedtls_").trim_start_matches("MBEDTLS_").to_owned())
+        // clean up generated static function suffix
+        let original_item_name = original_item_name.trim_end_matches("__extern");
+        // remove reductant prefix for code in mbedtls lib
+        Some(if original_item_name.starts_with("mbedtls_") {
+            original_item_name.trim_start_matches("mbedtls_").to_string()
+        } else if original_item_name.starts_with("MBEDTLS_") {
+            original_item_name.trim_start_matches("MBEDTLS_").to_string()
+        // remove reductant prefix for code in psa lib
+        } else if original_item_name.starts_with("psa_") {
+            original_item_name.trim_start_matches("psa_").to_string()
+        } else if original_item_name.starts_with("PSA_") {
+            original_item_name.trim_start_matches("PSA_").to_string()
+        } else {
+            original_item_name.to_string()
```

Comment:
the if statements are not necessary, you can do this:
```
original_item_name.trim_end_matches("__extern")
    .trim_start_matches("mbedtls_")
    .trim_start_matches("MBEDTLS_")
    .trim_start_matches("psa_")
    .trim_start_matches("PSA_")
    .to_owned()
```

Reply:
They are not identical. For example in case a function called `psa_mbedtls_foo` it will be changed `foo`.

The if statement ensure the trim only run once on each input